torch_em/transform/raw.py

```diff
@@ -36,10 +38,74 @@ def normalize_percentile(raw, lower=1.0, upper=99.0, axis=None, eps=1e-7):
     return normalize(raw, v_lower, v_upper, eps=eps)
 
 
+def normalize_torch(tensor, minval=None, maxval=None, eps=1e-7):
```

Comment:
Can we instead update `normalize` so that it also supports torch tensors? I think that the only thing that needs to be adapted is that the arguments to min and max need to be passed as normal arguments, not as kwargs.

Reply:
The kwargs are not the only problem, casting is also different and in general min()/max() behaves differently for numpy and torch. But anyway, I updated it now, so normalize() should now work for both.

Reply:
Thanks!